model/manifest.go

```diff
@@ -13,13 +13,41 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	PLUGIN_CONFIG_TYPE_TEXT      = "text"
+	PLUGIN_CONFIG_TYPE_BOOL      = "bool"
+	PLUGIN_CONFIG_TYPE_RADIO     = "radio"
+	PLUGIN_CONFIG_TYPE_DROPDOWN  = "dropdown"
+	PLUGIN_CONFIG_TYPE_GENERATED = "generated"
```

Comment:
Does this type also need a "regenerate help text" field? For example, the JIRA plugin has this below the regenerate button:

> Regenerates the secret for the webhook URL endpoint. Regenerating the secret invalidates your existing JIRA integrations.

Reply:
I'd prefer to keep it simpler and just have the single help text field but I'm only 1/5 on that

Reply:
It would be nice to support help text for each setting type.

Reply:
Alright, what about adding a field "secondary_help_text" to support any types that have a second spot for help text?

Reply:
I think I'd be in slight favor of just creating a new key for each help text ("regenerate_help_text", "other_thing_help_text", etc.) just to be more explicit. At the very least, it would make the schema read more intuitively (Someone reading it for the first time isn't gonna know what "secondary_help_text" means for each type.). Might also make our documentation clearer.

Reply:
I guess a bit of extra lines is worth the readability improvement